ONSdigital/dp-frontend-cookie-controller
# Controller populated and functional

### What

Controller now populated and functional
Further info can be found on the README.md file

### How to review

- Check that everything is functional and as expected
- Ensure that project criteria has been met, see trello tickets and Miro board for details
- Make sure there are no security issues in the project
- Make sure that there are enough tests and correct coverage

### Who can review

Anyone except me
